### PR TITLE
Improved Polish translation

### PR DESCRIPTION
**Pull request by m-aciek**
Style, punctuation

### CODE REVIEW

**Comment by haacked** [comment]
Thanks! Sorry for the late reply. Do you know other native Polish speakers who can double check your work here? That would be helpful to me as I don't speak Polish.

**Comment by m-aciek** [comment]
Sure, I will try to find someone to help and review those changes.

File: lang/pl/index.md [hunk 1]
```diff
@@ -1,42 +1,43 @@
...
 
-Mając do dyspozycji konkretny numer MAJOR.MINOR.PATCH, należy dokonać inkrementacji:
+Dla numeru wersji MAJOR.MINOR.PATCH, zwiększaj:
 
-1. MAJOR wersji, gdy dokonujesz zmian niekompatybilnych z API,
-1. MINOR wersji, gdy dodajesz nową funkcjonalność, która jest kompatybilna z poprzednimi wersjami,
-1. PATCH wersji, gdy dokonujesz naprawy błędu, która jest kompatybilna z poprzednimi wersjami.
+1. wersję MAJOR, gdy dokonujesz zmian niekompatybilnych z API,
+1. wersję MINOR, gdy dodajesz nową funkcjonalność, która jest kompatybilna z poprzednimi wersjami,
+1. wersję PATCH, gdy naprawiasz błąd w sposób kompatybilny z poprzednimi wersjami.
```
Comment (yisonPylkita) [inline lang/pl/index.md]: It should be `gdy naprawiasz błąd nie zrywając kompatybilności z poprzednimi wersjami.`

File: lang/pl/index.md [hunk 2]
```diff
@@ -1,42 +1,43 @@
...
 inkrementują wersję major.
 
-System ten został nazwany przeze mnie "Semantycznym Wersjonowaniem". W systemie tym numery wersji i sposób w jaki zostają zmienione wskazują na tworzący je kod oraz na to, co zostało zmodyfikowane od ostatniej wersji.
+System ten został nazwany przeze mnie „wersjonowaniem semantycznym”. W systemie tym numery wersji i sposób, w jaki zostają zmienione, wskazują na tworzący je kod oraz na to, co zostało zmodyfikowane od ostatniej wersji.
```
Comment (yisonPylkita) [inline lang/pl/index.md]: These commas are invalid here.

**Review by yisonPylkita — CHANGES_REQUESTED** [review]
Hi
I am a native polish speaker and colleague of @m-aciek.

Everything beside marked lines seems to be fine.

**Comment by m-aciek** [comment]
During last few days I've reworked the translation, making new translations for some parts of the document. I've split the lines to have width of 80, like in original document.

@yisonPylkita, could you review new changes and leave your feedback here again?

**Comment by yisonPylkita** [comment]
@m-aciek  Yes, of course. I will check changes tomorrow.

**Comment by yisonPylkita** [comment]
There seems to be a lot of changes. I will check them tomorrow.

**Comment by m-aciek** [comment]
Sure, thanks!

**Review by yisonPylkita — APPROVED** [review]
I have no objections to this change.  Most of the Readme was re-written and now reading it feels way better.

Great job @m-aciek

**Comment by haacked** [comment]
Thanks!

**Pull request merged**